components/Header.tsx

```diff
@@ -57,7 +57,7 @@ export const Header: React.FC<HeaderProps> = ({ userArrivedFromUioMobile = false
           <Navbar.Toggle aria-controls="basic-navbar-nav" />
           <Navbar.Collapse>
             <Nav>
-              <Nav.Link target="_blank" rel="noopener noreferrer" href={uioHomeLink}>
+              <Nav.Link rel="noopener noreferrer" href={uioHomeLink}>
```

Comment:
I assume that it's intentional that the link to UIO Home here _should not_ open in a new tab, but that the links above _should_?

Reply:
Yes